apps/extension/src/ui/hooks/ledger/useLedgerSubstrateChains.ts

```diff
@@ -0,0 +1,50 @@
+import { Chain } from "extension-core"
+import { useMemo } from "react"
+
+import { AddSubstrateLedgerAppType } from "@ui/domains/Account/AccountAdd/AccountAddLedger/context"
+
+import useChains from "../useChains"
+import { CHAIN_ID_TO_LEDGER_APP_NAME } from "./common"
+
+export type ChainWithLedgerApp = Chain & {
+  ledgerAppName?: string
+  supportedLedgerApps: AddSubstrateLedgerAppType[]
+}
+
+export const useLedgerSubstrateChains = () => {
+  const { chains } = useChains({ includeTestnets: false, activeOnly: true })
```

Comment:
wdyt about allowing the choice of all chains, not `activeOnly` ? 
I just had the problem where I have deactivated some chains but forgotten about it, and was confused why they were missing from the list.

Reply:
sure, will change it